gobblin-runtime/src/test/java/gobblin/GobblinRuntimeTest.java

```diff
@@ -0,0 +1,88 @@
+package gobblin;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import org.apache.commons.io.FileUtils;
+import org.testng.annotations.Test;
+
+import lombok.extern.slf4j.Slf4j;
+
+import gobblin.runtime.local.LocalJobLauncher;
+
+
+/**
+ * Created by adsharma on 11/22/16.
+ */
+
+@Slf4j
+public class GobblinRuntimeTest {
+  private static final String RESOURCE_DIR = "./gobblin-runtime/src/test/resources/gobblinRuntimeTest/";
+
+  public void invokeLocalJobLauncher(Properties properties)
+      throws Exception {
+    try (LocalJobLauncher localJobLauncher = new LocalJobLauncher(properties)) {
+      localJobLauncher.launchJob(null);
+    }
+  }
+
+  public Properties getJobProperties(String fileProperties)
+      throws IOException {
+    Properties jobProperties = new Properties();
+    InputStream inputStream = getClass().getClassLoader().getResourceAsStream(fileProperties);
+    jobProperties.load(inputStream);
+    return jobProperties;
+  }
+
+  public void cleanDir()
+      throws IOException {
+    FileUtils.cleanDirectory(new File(RESOURCE_DIR + "stateStore"));
+    FileUtils.cleanDirectory(new File(RESOURCE_DIR + "writerStaging"));
+    FileUtils.cleanDirectory(new File(RESOURCE_DIR + "writerOutput"));
+  }
+
+  /**
+   * This test is to validate that the skipped work units wont be passed to the publisher.
+   * @throws Exception
+   */
+
+  @Test
+  public void testSkippedWorkUnitsEvadePublisher()
+      throws Exception {
+    Properties jobProperties = getJobProperties("gobblinRuntimeTest/skipWorkUnitsTest.properties");
+    jobProperties.setProperty("data.publisher.type", "gobblin.skipWorkUnits.TestPublisher");
+    invokeLocalJobLauncher(jobProperties);
+    cleanDir();
```

Comment:
If all the tests are invoking it, should probably mark this as a tearDown/@AfterTest